openpype/tools/loader/model.py

```diff
@@ -361,10 +361,11 @@ def set_version(self, index, version):
             version_data.get("endFrame", None)
         )
 
+        handles_label = None
         handle_start = version_data.get("handleStart", None)
         handle_end = version_data.get("handleEnd", None)
         if handle_start is not None and handle_end is not None:
-            handles = "{}-{}".format(str(handle_start), str(handle_end))
+            handles_label = "{}-{}".format(str(handle_start), str(handle_end))
```

Comment:
Could we put the `handles_label = None` into the else statement instead? It's much more explicit then that it's THIS or THAT than the current implementation.

So basically bringing back the logic from before #4746 but setting the value to None.

Reply:
> Could we put the handles_label = None into the else statement instead?

I would say this is more explicit in a way that the variable must be 100% available, so it's clear it must be available in the scope.

Reply:
Fine with me.